cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,184 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Article', () => {
-  before(() => {
+import { generateNewArticle } from "../support/generateNewArticle";
+import faker from 'faker';
 
+describe('Create new article', () => {  
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.registerNewUser().then(user => {
+      cy.login(user).then(() => user);
+    }).as('user');
+
+    cy.visit(`#/editor`);
+  });
+
+  it(`The Creating new article page 
+    should contains 'Article Title' field`, () => {
+      cy.findByPlaceholder('Article Title').should('exist');
+  });
+
+  it(`The Creating new article page 
+  should contains 'About artice' field`, () => {
+    cy.findByPlaceholder(`What's this article about?`).should('exist');
+  });
+
+  it(`The Creating new article page 
+  should contains 'Write your artice' field`, () => {
+    cy.findByPlaceholder(`Write your article (in markdown)`).should('exist');
+  });
+
+  it(`The Creating new article page 
+  should contains 'Enter tags' field`, () => {
+    cy.findByPlaceholder(`Enter tags`).should('exist');
+  });
+
+  it(`The Creating new article page 
+  should contains 'Publish Article' button`, () => {
+    cy.contains('button', 'Publish Article').should('exist');
+  });
+
+  it('The user can create a new article with filled all field', () => {
+    const { articleTitle, articleDescription, articleBody, articleTags }
+      = generateNewArticle();
+
+    cy.findByPlaceholder('Article Title').type(articleTitle);
+    cy.findByPlaceholder(`What's this article about?`).type(articleDescription);
+    cy.findByPlaceholder(`Write your article (in markdown)`).type(articleBody);
+    cy.findByPlaceholder(`Enter tags`).type(`${articleTags}{enter}`);
+    cy.contains('button', 'Publish Article').click();
+    cy.wait(2000);
+    cy.get('h1').should('contain.text', articleTitle);
+    cy.get('p').should('contain.text', articleBody);
+  });
+
+  it('The user can create new article using only title', () => {
```

Comment:
Revise the decision table from the Test Design Techniques HW. Use the same approach as there. Also, it will be great to test the inputs using EC and BV